x/feegrant/keeper/grpc_query.go

```diff
@@ -93,3 +93,42 @@ func (q Keeper) Allowances(c context.Context, req *feegrant.QueryAllowancesReque
 
 	return &feegrant.QueryAllowancesResponse{Allowances: grants, Pagination: pageRes}, nil
 }
+
+// IssuedAllowances queries all the allowances granted by the given granter
+func (q Keeper) IssuedAllowances(c context.Context, req *feegrant.QueryIssuedAllowancesRequest) (*feegrant.QueryIssuedAllowancesResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "invalid request")
+	}
+
+	granterAddr, err := sdk.AccAddressFromBech32(req.Granter)
+	if err != nil {
+		return nil, err
+	}
+
+	ctx := sdk.UnwrapSDKContext(c)
+
+	var grants []*feegrant.Grant
+
+	store := ctx.KVStore(q.storeKey)
+	pageRes, err := query.Paginate(store, req.Pagination, func(key []byte, value []byte) error {
+		var grant feegrant.Grant
+
+		granter, _ := feegrant.ParseAddressesFromFeeAllowanceKey(key)
+		if !granter.Equals(granterAddr) {
+			return nil
+		}
+
+		if err := q.cdc.Unmarshal(value, &grant); err != nil {
+			return err
+		}
+
+		grants = append(grants, &grant)
+		return nil
+	})
```

Comment:
I think iterating all the store keys might not be a better idea (it can be possible this store can be big).
cc: @aaronc , @amaurym

Reply:
Yes the more efficient solution would be to have a secondary index and I think in the future we may want to have the `feegrant` module use orm. For now I think this suffices due to the following reasons:
- The range scan is only parsing the keys and not the values to determine whether the allowance comes from the granter
- I can't imagine chains having a state size greater than a few hundred entries
- This is just a query endpoint (only affects individual nodes) and can be turned off from the public if need be.
- Most use cases will be for individuals checking grants on their own machines. Most "commercial" level products will be using something like a postgres indexer for serving `feegrant` information